HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -47,6 +47,105 @@ def customiseFor14833(process):
+            doubletLabel = label + "HitPairs"
+            tripletLabel = label + "HitTriplets"
+
+        # Construct new producers
+        regionProducer = _globalTrackingRegionFromBeamSpot.clone()
+        regionProducer.RegionPSet = producer.RegionFactoryPSet.RegionPSet
+
+        clusterCheckProducer = _clusterCheckerEDProducer.clone()
+        _copy(producer.ClusterCheckPSet, clusterCheckProducer)
+
+        doubletProducer = _hitPairEDProducer.clone(
+            seedingLayers = producer.OrderedHitsFactoryPSet.SeedingLayers.value(),
+            trackingRegions = regionLabel,
+            clusterCheck = clusterCheckLabel,
+            produceIntermediateHitDoublets = True,
+        )
+
+        tripletProducer = _pixelTripletHLTEDProducer.clone(
+            doublets = doubletLabel,
+            produceSeedingHitSets = True,
+        )
+        _copy(producer.OrderedHitsFactoryPSet.GeneratorPSet, tripletProducer, skip=["ComponentName"])
+
+        seedCreatorPSet = producer.SeedCreatorPSet
+        if hasattr(seedCreatorPSet, "refToPSet_"):
+            seedCreatorPSet = getattr(process, seedCreatorPSet.refToPSet_.value())
+
+        seedProducer = _seedCreatorFromRegionConsecutiveHitsEDProducer.clone(
+            seedingHitSets = tripletLabel,
+        )
+        _copy(seedCreatorPSet, seedProducer, skip=["ComponentName"])
+
+        # Set new producers to process
+        setattr(process, regionLabel, regionProducer)
+        setattr(process, clusterCheckLabel, clusterCheckProducer)
+        setattr(process, doubletLabel, doubletProducer)
+        setattr(process, tripletLabel, tripletProducer)
+        modifier.toReplaceWith(producer, seedProducer)
+
+        # Modify sequences (also paths to be sure, altough in practice
+        # the seeding modules should be only in sequences in HLT?)
+        for seqs in [process.sequences_(), process.paths_()]:
+            for seqName, seq in seqs.iteritems():
+                # Is there really no simpler way to add
+                # regionProducer+doubletProducer+tripletProducer
+                # before producer in the sequence?
+                #
+                # cms.Sequence.replace() would look much simpler, but
+                # it traverses the contained sequences too, leading to
+                # multiple replaces as we already loop over all
+                # sequences of a cms.Process, and also expands the
+                # contained sequences if a replacement occurs there.
+                try:
+                    index = seq.index(producer)
+                except:
+                    continue
+
+                # Inserted on reverse order, succeeding module will be
+                # inserted before preceding one
+                seq.insert(index, tripletProducer)
+                seq.insert(index, doubletProducer)
+                seq.insert(index, clusterCheckProducer)
+                seq.insert(index, regionProducer)
+
+    return process
+
```

Comment:
As I noted in the description, the above is just for demonstration and I'd leave it out of the first PR in order to manage with a single customize function handling all cases.